app/controllers/signs_controller.rb

```diff
@@ -4,11 +4,11 @@ class SignsController < ApplicationController
   layout 'admin', except: [:play]
 
   skip_before_action :authenticate_user!, only: [:play, :poll]
-  
+
   before_action :set_sign, only: [:remove_slide, :poll, :show, :edit, :update, :destroy, :play, :settings, :order]
   before_action :set_search_filters, only: [:index]
 
-  load_and_authorize_resource
+  load_and_authorize_resource except: [:play, :poll]
```

Comment:
One mystery here if anyone wants to take a stab at explaining it: `play` endpoint worked even without this exception. I added it nonetheless for the sake of clarity. But any idea why? Something to do with CanCan?